pkg/apis/kubernetes/v1/kubeconfig_types.go

```diff
@@ -0,0 +1,5 @@
+package v1
+
+type kubeconfig struct {
+	Cluster string `json:"cluster" anxcloud:"identifier"`
+}
```

Comment:
I think this could go into `kubeconfig_helper.go` as well, being an internal type and all